modules/steam.js

```diff
@@ -0,0 +1,117 @@
+var keys    = require('../util/keys');
+var SteamApi = require('steam-api');
+var request = require('request');;
+
+var app = new SteamApi.App(keys.STEAM.KEY);
+var player = new SteamApi.Player(keys.STEAM.KEY);
+var user = new SteamApi.User(keys.STEAM.KEY);
```

Comment:
Ditto

Reply:
Also let's ax semicolons. They seem to be a bit of a struggle too; you put two on line 3.